src/dlgAboutDialog.cpp

```diff
@@ -94,4 +98,696 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
+                   "<p><span style=\"color:#bc8942;\"><big><b>Ahmed Charles</b></big></span> (<span style=\"color:#40b040;\">ahmedcharles</span> <span style=\"color:#0000ff;\">acharles@outlook.com</span>) contributions to the Travis integration, CMake and Visual C++ build, a lot of code quality and memory management improvements.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><big><b>Chris Mitchell</b></big></span> (<span style=\"color:#40b040;\">Chris7</span> <span style=\"color:#0000ff;\">chrismudlet@gmail.com</span>) has developed a shared module system that allows script packages to be shared among profiles, a UI for viewing Lua variables, improvements in the mapper and all around.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Ben Carlsen</b></span> (<span style=\"color:#0000ff;\">arkholt@gmail.com</span>) has developed the first version of our Mac OSX installer. He is the former maintainer of the Mac version of Mudlet.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Ben Smith</b></span> () joined in December 2009 though he's been around much longer. Contributed to the Lua API and is the former maintainer of the Lua API.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Blaine von Roeder</b></span> () joined in December 2009. He has contributed to the Lua API, submitted small bugfix patches and has helped with release management of 1.0.5.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Bruno Bigras</b></span> (<span style=\"color:#0000ff;\">bruno@burnbox.net</span>) developed the original cmake build script and he has committed a number of patches.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Carter Dewey</b></span> (<span style=\"color:#0000ff;\">eldarerathis@gmail.com</span>) contributions to the Lua API.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Erik Pettis</b></span> (<span style=\"color:#40b040;\">Oneymus</span>) developed the Vzyor GUI Manager for Mudlet.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>\"ItsTheFae\"</b></span> (<span style=\"color:#40b040;\">Kae</span>) someone who has worked wonders in rejuventating our Website in 2017 but who prefers a little anonymity - if you are a <i>SpamBot</i> you will not get onto our Fora now. They have also made some useful C++ core code contributions and we look forward to future reviews on and work in that area.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>\"dicene\"</b></span> (<span style=\"color:#40b040;\">Ian</span>) joining us 2017 they have given us some useful C++ and Lua contributions.</p>\n"
```

Comment:
He's listed in [copyright](https://github.com/Mudlet/Mudlet/blob/development/src/mudlet.cpp#L6) by name and email - could do the same here :)

Reply:
Okay - I'll recheck those details, note that I'm including Github Ids in the same colour as the Github links at the top of the page... 😁

Reply:
Fixed.